app/_src/production/dp-config/transparent-proxying.md

```diff
@@ -53,6 +54,7 @@ The host that will run the `kuma-dp` process in transparent proxying mode needs
    iptables --version
    # iptables v1.8.7 (legacy)
    ```
+{% endif_version %}
 
 2. Create a new dedicated user on the machine.
```

Comment:
If_version less than 2.4.x.

We'll get the title number `2` here. 
I recommend to use title number `1` for `Create a new dedicated user on the machine.`
And the upper condition line use title `prerequisite` or something else.

Reply:
We won't :-) It doesn't matter what is the number here, we can even set all items to have a `1` as a list item number, as renderer will still render it correctly, ref: https://deploy-preview-1787--kuma.netlify.app/docs/2.7.x/production/dp-config/transparent-proxying/#setting-up-the-service-host